docs/reference/modules/ROOT/pages/queries.adoc

```diff
@@ -3,27 +3,37 @@
+Crux automatically indexes the top-level fields in all documents, supporting efficient ad-hoc joins and retrievals.
+Crux is both _immutable_ and _bitemporal_, which are both required characteristics for a safe schemaless database.
 
 Crux is also a graph database.
 The central characteristic of a graph database is that it can support arbitrary-depth graph queries (recursive traversals) very efficiently by default, without any need for schema-level optimisations.
-Crux gives you the ability to construct graph queries via a Datalog query language and uses graph-friendly indexes to provide a powerful set of querying capabilities.
-Additionally, when Crux's indexes are deployed directly alongside your application you are able to easily blend Datalog and code together to construct highly complex graph algorithms.
+Graph queries are possible through the Crux dialect of the Datalog query language.
 
 //This page walks through many of the more interesting queries that run as part of Crux's default test suite.
 //See the https://github.com/juxt/crux/blob/master/crux-test/test/crux/query_test.clj[`Crux query test file`] for the full suite of query tests and how each test listed below runs in the wider context.
 
-Extensible Data Notation (edn) is used as the data format for the public Crux APIs.
-To gain an understanding of edn see xref:tutorials::essential-edn.adoc[Essential EDN for Crux].
+=== SQL
 
-Note that all Crux Datalog queries run using a point-in-time view of the database which means the query capabilities and patterns presented in this section are not aware of valid times or transaction times.
+Crux 1.x supports both Datalog and SQL, but SQL support is provided through a module.
+To query Crux using SQL, read the xref:reference::sql.adoc[SQL module documentation].
 
-A Datalog query consists of a set of variables and a set of clauses. The result
-of running a query is a result set of the possible combinations of values that
-satisfy all of the clauses at the same time. These combinations of values are
-referred to as "tuples".
+=== Datalog
+
+Extensible Data Notation (edn) is used as the data format for the Crux Datalog query APIs.
+EDN is a simple format.
+To understand how edn works, read the brief description at http://edn-format.org/[http://edn-format.org].
+
+To understand how Datalog works, read the _Learn Crux Datalog Today_ tutorial from the https://www.opencrux.com/tutorials/[Tutorials page].
+
+////
+Can we remove these 3 paragraphs? -sd
+
+A Datalog query consists of a set of variables and a set of clauses.
+The result of running a query is a result set of the possible combinations of values that
+satisfy all of the clauses at the same time.
+These combinations of values are referred to as "tuples".
 
 The possible values within the result tuples are derived from your database of
 documents. The documents themselves are represented in the database indexes as
```

Comment:
Ideally this nuance would be captured in a model page, that users should study before look at queries. Without this paragraph though, the topic only gets a brief mention under "Maps and Vectors in data" so it risks making things more confusing